client/src/App.css

```diff
@@ -1,42 +1,174 @@
-.App {
-  text-align: center;
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Inter';
+  font-style: normal;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
+.App {
+  width: 100vw;
+  height: 100vh;
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+  overflow-y: hidden;
+  overflow-x: hidden;
+
+  background: linear-gradient(
+      180deg,
+      rgba(255, 255, 255, 0) 0%,
+      rgba(221, 204, 231, 0.21875) 38.02%,
+      #621790 100%
+    ),
+    #935ebf;
+
+  color: #ffffff;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+/* TITLE SECTION */
+.title {
   display: flex;
   flex-direction: column;
-  align-items: center;
+
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  align-items: center;
+  margin: 0 auto;
+
+  text-align: center;
+  padding-top: 10%;
+}
+.title-text {
+  font-size: 64px;
+  font-style: bold;
+  letter-spacing: 0.1em;
+  margin-bottom: 24px;
+  text-shadow: 0px 7px 10px rgba(0, 0, 0, 0.25);
+}
+hr.line {
+  top: 10px;
+  width: 851px;
+  height: 0px;
+  border: 2px solid #ffe76a;
+  border-radius: 100px;
+  margin-bottom: 24px;
+}
+.subtitle {
+  font-size: 20px;
+  letter-spacing: 0.05em;
```

Comment:
usually letter spacing isn't added unless it's really needed, it's okay to keep in though